taxonomies/ingredients_processing.txt

```diff
@@ -1645,6 +1684,122 @@ wikidata:en: Q7601513
 #fr:germe, gluten, amidon
 #hr:klica, gluten, škrob
 
+en: condensed
+af: Kondens
+ar: المبخر
+az: Qatılaşdırılmış
+be: згушчанае
+bg: кондензирано
+ca: condensada
+cs: kondenzované
+da: kondenserad, kondenseret
```

Comment:
“kondenserad” is not Danish. Is this a Swedish ingredient list that got conflated with Danish at some point?
```suggestion
da: kondenseret
```